app/webpacker/components/Panel/pages/AnonymizationScriptPage/index.jsx

```diff
@@ -1,25 +1,53 @@
-import React from 'react';
-import WcaSearch from '../../../SearchWidget/WcaSearch';
+import React, { useEffect } from 'react';
+import { FormField, FormGroup, Radio } from 'semantic-ui-react';
+import { IdWcaSearch } from '../../../SearchWidget/WcaSearch';
 import SEARCH_MODELS from '../../../SearchWidget/SearchModel';
 import useInputState from '../../../../lib/hooks/useInputState';
-import AnonymizationTicketWorkbench from '../../../Tickets/TicketWorkbenches/AnonymizationTicketWorkbench';
+import AnonymizationTicketWorkbenchForWrt from '../../../Tickets/TicketWorkbenches/AnonymizationTicketWorkbenchForWrt';
+
+const MODEL_NAME = {
+  [SEARCH_MODELS.user]: 'User',
+  [SEARCH_MODELS.person]: 'Person',
+};
 
 export default function AnonymizationScriptPage() {
-  const [user, setUser] = useInputState();
+  const [model, setModel] = useInputState(SEARCH_MODELS.user);
+  const [searchInput, setSearchInput] = useInputState();
+
+  useEffect(() => setSearchInput(null), [model, setSearchInput]);
 
   return (
     <>
-      <WcaSearch
-        label="Enter the user to anonymize"
-        model={SEARCH_MODELS.user}
+      <FormGroup grouped>
+        <div>Select where to search for</div>
+        {[SEARCH_MODELS.user, SEARCH_MODELS.person].map((searchModel, index) => (
+          <FormField key={searchModel}>
+            <Radio
+              label={MODEL_NAME[index]}
+              value={searchModel}
+              checked={model === searchModel}
+              onChange={setModel}
```

Comment:
You could both call `setModel` and reset the search input here. This would allow you to remove the effect above. Generally, if you can do something without an effect, then you should.

Reply:
Okay done